### PR TITLE
update registry.redhat.io instruction

### PR DESCRIPTION
**Pull request by leena-jawale**

### CODE REVIEW

**Review by zkayyali812 — APPROVED** [review]

File: aap-common/proc-aap-pull-command-container-image.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ Pull the Docker image for the Ansible on Clouds operational container with the s
 
 [NOTE]
 ====
-Before Pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before Pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline aap-common/proc-aap-pull-command-container-image.adoc:9]: ensure you are logged in

File: stories/topics/con-aws-pull-container-image.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ Pull the Docker image for the Ansible on Clouds operational container which alig
 
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/con-aws-pull-container-image.adoc:9]: ensure you are logged in

File: stories/topics/con-aws-pull-deploy-container-image.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@ Pull the Docker image for the Ansible on Clouds operational container with the s
 
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/con-aws-pull-deploy-container-image.adoc:9]: as before

File: stories/topics/con-gcp-use-container-image.adoc [hunk 4]
```diff
@@ -4,6 +4,16 @@
 
 Pull the Docker image for the Ansible on Clouds operational container which aligns with the version of your foundation deployment.
 
+[NOTE]
+====
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/con-gcp-use-container-image.adoc:9]: as before

File: stories/topics/proc-aws-backup-platform-stack.adoc [hunk 5]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com. 
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-aws-backup-platform-stack.adoc:10]: as before

File: stories/topics/proc-aws-from-stack-pull-container-image.adoc [hunk 6]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-aws-from-stack-pull-container-image.adoc:10]: as before

File: stories/topics/proc-aws-set-container-image.adoc [hunk 7]
```diff
@@ -10,7 +10,7 @@ For example, if your foundation deployment version is 2.3.20230221-00, pull the
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com. 
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-aws-set-container-image.adoc:13]: as before

File: stories/topics/proc-aws-upgrade-pull-container-image.adoc [hunk 8]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com. 
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-aws-upgrade-pull-container-image.adoc:10]: as before

File: stories/topics/proc-gcp-backup-container-image.adoc [hunk 9]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com. 
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-container-image.adoc:10]: as before

File: stories/topics/proc-gcp-from-stack-pull-container-image.adoc [hunk 10]
```diff
@@ -8,7 +8,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-gcp-from-stack-pull-container-image.adoc:11]: as before

File: stories/topics/proc-gcp-pull-backup-container-image.adoc [hunk 11]
```diff
@@ -8,7 +8,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-gcp-pull-backup-container-image.adoc:11]: as before

File: stories/topics/proc-gcp-upgrade-pull-container-image.adoc [hunk 12]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-gcp-upgrade-pull-container-image.adoc:10]: as before

**Review by ianf77 — COMMENTED** [review]
A few comments

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by leena-jawale** [comment]
@ianf77 Please merge this PR when you get a chance.

**Pull request merged**